_pages/projects/2020-auto-indy.md

```diff
@@ -0,0 +1,12 @@
+---
+layout: project_extended
+title: Indy Autonomous Challenge
+permalink: /projects/2020-auto-indy
+image: /assets/images/projects/2020/project-auto-indy.jpg
+leaders:  Nayana Suvarna, Prem Bharatia, Andrew Saba, Joshua Spisak
+meeting_time: Saturday at 12pm
+albumID: 72157666933540901
+project_tag: 2020-auto-indy
+---
+
```

Comment:
Actual content from the project layout page should be inserted here. All formatting should use Markdown. I do believe that Jekyll does extend Markdown a tiny bit and it is possible to use html tag as necessary.

Reply:
On the upside of using markdown instead of raw HTML, it is ever slightly faster to write content.